mainstyle.css

```diff
@@ -0,0 +1,96 @@
+* {
+  width: 100%;
+  height: 100%;
```

Comment:
using height:100% with the * selector is generally not the best idea.   The * selector applies a given style to every single element in the page.  A better use would be using html, body as follows
```CSS
html, body {
    height: 100%;
}
```